### ISSUE TITLE
Snapcraft linter reports missing libraries

### ISSUE THREAD

**Issue by brlin-tw** [brlin-tw/OpenSyobonAction-snap#10]
Refer to the following Snapcraft output:

```txt
+ snapcraftctl prime
The 'main' part is missing libraries that are not included in the snap or base. They can be satisfied by adding the following entries to the existing stage-packages for this part:
- libgl1
- libglu1-mesa
- libglx0
This part is missing libraries that cannot be satisfied with any available stage-packages known to snapcraft:
- libglut.so.3
These dependencies can be satisfied via additional parts or content sharing. Consider validating configured filesets if this dependency was built.
```

We need to determine whether the libraries are really missing, or it's just a false-positive.

**Comment by brlin-tw** [comment]
The application launches without these libraries, so it's more likely a false-positive...

**Comment by brlin-tw** [comment]
libglut.so.3 library is provided by the freeglut3 package.

**Comment by brlin-tw** [comment]
Fixed in revision 6f73406.
